README.md

```diff
@@ -296,5 +296,5 @@ After a successful review by a maintainer, the PR may be merged successfully, an
 - [Git](https://try.github.io)
 - [PyCharm](https://www.jetbrains.com/help/pycharm/quick-start-guide.html)
 - [Pipenv](https://pipenv.readthedocs.io)
-- [discord.py rewrite](https://discordpy.readthedocs.io/en/rewrite)
+- [discord.py](https://discordpy.readthedocs.io/en/latest/)
```

Comment:
```suggestion
- [discord.py](https://discordpy.readthedocs.io/en/v1.1.0/)
```
#201 is going to pin the d.py dependency to `v1.1.0` so the docs should point to this rather than latest. We can update the link as we update the dependency pin.